### ISSUE TITLE
Code to run when idle

### ISSUE THREAD

**Issue by wilburforce666** [dahall/TaskScheduler#56]
How can I use this to make a program run when no input is made for X amount of time?

**Comment by dahall** [comment]
I am assuming that you mean when the system is idle for X amount of time. If so, you can use an IdleTrigger. For example, let's say you want to run the program "wakeup.exe" after the system has been idle 10 minutes.
```c#
// Create a new task definition and assign properties
TaskDefinition td = TaskService.Instance.NewTask();

// Tell the task to wait 10 minutes after an idle condition before starting
td.Settings.IdleSettings.IdleDuration = TimeSpan.FromMinutes(10);

// Add a trigger that launches the task when the machine is idle
td.Triggers.Add(new IdleTrigger());

// Create an action for the program
td.Actions.Add("Wakeup.exe");

// Register the task in the root folder
TaskService.Instance.RootFolder.RegisterTaskDefinition(@"WakeupTask", td);
```
